docs/docs/image-filters/runtime-shader.md

```diff
@@ -0,0 +1,39 @@
+---
+id: runtime-shader
+title: Runtime Shader
+sidebar_label: Runtime Shader
+slug: /image-filters/runtime-shader
+---
+
+The `RuntimeShader` image filter allows you to provide your own [Skia Shader](/docs/shaders/overview) as an image filter.
+
+| Name      | Type              |  Description                     |
+|:----------|:------------------|:---------------------------------|
+| source    | `SkRuntimeEffect` | Shader to use as an image filter |
+
+## Example
+
+```tsx twoslash
+import {Canvas, Text, RuntimeShader, Skia, Group, Circle} from "@shopify/react-native-skia";
+
+const source = Skia.RuntimeEffect.Make(`
+uniform shader image;
+
+half4 main(float2 xy) {
+  return image.eval(xy).rbga;
+}
+`)!;
+
+export const RuntimeShaderDemo = () => {
+  const r = 128;
+  return (
+    <Canvas style={{ flex: 1 }}>
+      <Group>
+         <RuntimeShader source={source} />
+         <Circle cx={r} cy={r} r={r} color="lightblue" />
```

Comment:
Maybe the docs are a bit short here? Might not be totally clear what the Runtime shader does here and what it affects. Later in the example we also use the following 

```tsx
<>
  <Paint ref={paint}>
    <RuntimeShader source={source} />
  </Paint>
  <Group layer={paint}>{children}</Group>
</>
```

Which also might be good to explain in more detail?